_posts/2021-05-26-registry-security.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Securing Apicurio Registry 2.0 using Keycloak"
+date:   2021-05-28 12:00:00
```

Comment:
Can you rename the file and the date to be consistent?

Reply:
Whoops, sorry.